.github/workflows/ci.yml

```
@@ -490,7 +490,9 @@ jobs:
 
         if [ "${{ github.event_name }}" == "pull_request" ]; then
           # Invoked from a PR - get the PR body directly
-          MESSAGE="${{ github.event.pull_request.body }}"
+          read -r -d '' MESSAGE <<'EOF'
+            ${{ github.event.pull_request.body }}
+        EOF
```

Indent to the same level as the rest of the `if` body?

```suggestion
          EOF
```

---

Testing locally, it looks like the EOF needs to be at the start of the line.

---

Ack